[2025-04-16T20:27:38.419+0000] {processor.py:161} INFO - Started process (PID=23) to work on /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:27:38.421+0000] {processor.py:830} INFO - Processing file /root/airflow/dags/financial_trading_dag.py for tasks to queue
[2025-04-16T20:27:38.424+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:27:38.423+0000] {dagbag.py:545} INFO - Filling up the DagBag from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:27:38.428+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:27:38.428+0000] {dagbag.py:355} ERROR - Failed to import: /root/airflow/dags/financial_trading_dag.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/airflow/dags/financial_trading_dag.py", line 6, in <module>
    from src.data.get_stock import download_stock_data
ModuleNotFoundError: No module named 'src'
[2025-04-16T20:27:38.429+0000] {processor.py:842} WARNING - No viable dags retrieved from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:27:38.486+0000] {processor.py:183} INFO - Processing /root/airflow/dags/financial_trading_dag.py took 0.075 seconds
[2025-04-16T20:28:12.359+0000] {processor.py:161} INFO - Started process (PID=28) to work on /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:28:12.370+0000] {processor.py:830} INFO - Processing file /root/airflow/dags/financial_trading_dag.py for tasks to queue
[2025-04-16T20:28:12.372+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:28:12.371+0000] {dagbag.py:545} INFO - Filling up the DagBag from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:28:12.374+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:28:12.374+0000] {dagbag.py:355} ERROR - Failed to import: /root/airflow/dags/financial_trading_dag.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/airflow/dags/financial_trading_dag.py", line 6, in <module>
    from src.data.get_stock import download_stock_data
ModuleNotFoundError: No module named 'src'
[2025-04-16T20:28:12.375+0000] {processor.py:842} WARNING - No viable dags retrieved from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:28:12.393+0000] {processor.py:183} INFO - Processing /root/airflow/dags/financial_trading_dag.py took 0.039 seconds
[2025-04-16T20:28:42.542+0000] {processor.py:161} INFO - Started process (PID=29) to work on /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:28:42.552+0000] {processor.py:830} INFO - Processing file /root/airflow/dags/financial_trading_dag.py for tasks to queue
[2025-04-16T20:28:42.553+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:28:42.553+0000] {dagbag.py:545} INFO - Filling up the DagBag from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:28:42.557+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:28:42.555+0000] {dagbag.py:355} ERROR - Failed to import: /root/airflow/dags/financial_trading_dag.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/airflow/dags/financial_trading_dag.py", line 6, in <module>
    from src.data.get_stock import download_stock_data
ModuleNotFoundError: No module named 'src'
[2025-04-16T20:28:42.557+0000] {processor.py:842} WARNING - No viable dags retrieved from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:28:42.572+0000] {processor.py:183} INFO - Processing /root/airflow/dags/financial_trading_dag.py took 0.035 seconds
[2025-04-16T20:29:12.688+0000] {processor.py:161} INFO - Started process (PID=30) to work on /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:29:12.689+0000] {processor.py:830} INFO - Processing file /root/airflow/dags/financial_trading_dag.py for tasks to queue
[2025-04-16T20:29:12.690+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:29:12.690+0000] {dagbag.py:545} INFO - Filling up the DagBag from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:29:12.693+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:29:12.692+0000] {dagbag.py:355} ERROR - Failed to import: /root/airflow/dags/financial_trading_dag.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/airflow/dags/financial_trading_dag.py", line 6, in <module>
    from src.data.get_stock import download_stock_data
ModuleNotFoundError: No module named 'src'
[2025-04-16T20:29:12.693+0000] {processor.py:842} WARNING - No viable dags retrieved from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:29:12.710+0000] {processor.py:183} INFO - Processing /root/airflow/dags/financial_trading_dag.py took 0.027 seconds
[2025-04-16T20:29:43.825+0000] {processor.py:161} INFO - Started process (PID=31) to work on /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:29:43.835+0000] {processor.py:830} INFO - Processing file /root/airflow/dags/financial_trading_dag.py for tasks to queue
[2025-04-16T20:29:43.836+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:29:43.836+0000] {dagbag.py:545} INFO - Filling up the DagBag from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:29:43.839+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:29:43.838+0000] {dagbag.py:355} ERROR - Failed to import: /root/airflow/dags/financial_trading_dag.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/airflow/dags/financial_trading_dag.py", line 6, in <module>
    from src.data.get_stock import download_stock_data
ModuleNotFoundError: No module named 'src'
[2025-04-16T20:29:43.839+0000] {processor.py:842} WARNING - No viable dags retrieved from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:29:43.855+0000] {processor.py:183} INFO - Processing /root/airflow/dags/financial_trading_dag.py took 0.034 seconds
[2025-04-16T20:30:14.286+0000] {processor.py:161} INFO - Started process (PID=32) to work on /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:30:14.312+0000] {processor.py:830} INFO - Processing file /root/airflow/dags/financial_trading_dag.py for tasks to queue
[2025-04-16T20:30:14.344+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:30:14.331+0000] {dagbag.py:545} INFO - Filling up the DagBag from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:30:14.430+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:30:14.404+0000] {dagbag.py:355} ERROR - Failed to import: /root/airflow/dags/financial_trading_dag.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/airflow/dags/financial_trading_dag.py", line 6, in <module>
    from src.data.get_stock import download_stock_data
ModuleNotFoundError: No module named 'src'
[2025-04-16T20:30:14.446+0000] {processor.py:842} WARNING - No viable dags retrieved from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:30:14.857+0000] {processor.py:183} INFO - Processing /root/airflow/dags/financial_trading_dag.py took 0.690 seconds
[2025-04-16T20:30:45.707+0000] {processor.py:161} INFO - Started process (PID=33) to work on /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:30:45.717+0000] {processor.py:830} INFO - Processing file /root/airflow/dags/financial_trading_dag.py for tasks to queue
[2025-04-16T20:30:45.719+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:30:45.718+0000] {dagbag.py:545} INFO - Filling up the DagBag from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:30:45.721+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:30:45.721+0000] {dagbag.py:355} ERROR - Failed to import: /root/airflow/dags/financial_trading_dag.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/airflow/dags/financial_trading_dag.py", line 6, in <module>
    from src.data.get_stock import download_stock_data
ModuleNotFoundError: No module named 'src'
[2025-04-16T20:30:45.721+0000] {processor.py:842} WARNING - No viable dags retrieved from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:30:45.737+0000] {processor.py:183} INFO - Processing /root/airflow/dags/financial_trading_dag.py took 0.034 seconds
[2025-04-16T20:31:16.114+0000] {processor.py:161} INFO - Started process (PID=35) to work on /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:31:16.124+0000] {processor.py:830} INFO - Processing file /root/airflow/dags/financial_trading_dag.py for tasks to queue
[2025-04-16T20:31:16.126+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:31:16.126+0000] {dagbag.py:545} INFO - Filling up the DagBag from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:31:16.129+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:31:16.128+0000] {dagbag.py:355} ERROR - Failed to import: /root/airflow/dags/financial_trading_dag.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/airflow/dags/financial_trading_dag.py", line 6, in <module>
    from src.data.get_stock import download_stock_data
ModuleNotFoundError: No module named 'src'
[2025-04-16T20:31:16.129+0000] {processor.py:842} WARNING - No viable dags retrieved from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:31:16.144+0000] {processor.py:183} INFO - Processing /root/airflow/dags/financial_trading_dag.py took 0.035 seconds
[2025-04-16T20:31:46.826+0000] {processor.py:161} INFO - Started process (PID=36) to work on /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:31:46.827+0000] {processor.py:830} INFO - Processing file /root/airflow/dags/financial_trading_dag.py for tasks to queue
[2025-04-16T20:31:46.828+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:31:46.828+0000] {dagbag.py:545} INFO - Filling up the DagBag from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:31:46.830+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:31:46.830+0000] {dagbag.py:355} ERROR - Failed to import: /root/airflow/dags/financial_trading_dag.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/airflow/dags/financial_trading_dag.py", line 6, in <module>
    from src.data.get_stock import download_stock_data
ModuleNotFoundError: No module named 'src'
[2025-04-16T20:31:46.831+0000] {processor.py:842} WARNING - No viable dags retrieved from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:31:46.849+0000] {processor.py:183} INFO - Processing /root/airflow/dags/financial_trading_dag.py took 0.028 seconds
[2025-04-16T20:32:17.969+0000] {processor.py:161} INFO - Started process (PID=37) to work on /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:32:17.970+0000] {processor.py:830} INFO - Processing file /root/airflow/dags/financial_trading_dag.py for tasks to queue
[2025-04-16T20:32:17.972+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:32:17.971+0000] {dagbag.py:545} INFO - Filling up the DagBag from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:32:17.974+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:32:17.974+0000] {dagbag.py:355} ERROR - Failed to import: /root/airflow/dags/financial_trading_dag.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/airflow/dags/financial_trading_dag.py", line 6, in <module>
    from src.data.get_stock import download_stock_data
ModuleNotFoundError: No module named 'src'
[2025-04-16T20:32:17.974+0000] {processor.py:842} WARNING - No viable dags retrieved from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:32:17.991+0000] {processor.py:183} INFO - Processing /root/airflow/dags/financial_trading_dag.py took 0.027 seconds
[2025-04-16T20:32:49.072+0000] {processor.py:161} INFO - Started process (PID=38) to work on /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:32:49.073+0000] {processor.py:830} INFO - Processing file /root/airflow/dags/financial_trading_dag.py for tasks to queue
[2025-04-16T20:32:49.074+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:32:49.074+0000] {dagbag.py:545} INFO - Filling up the DagBag from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:32:49.077+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:32:49.077+0000] {dagbag.py:355} ERROR - Failed to import: /root/airflow/dags/financial_trading_dag.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/airflow/dags/financial_trading_dag.py", line 6, in <module>
    from src.data.get_stock import download_stock_data
ModuleNotFoundError: No module named 'src'
[2025-04-16T20:32:49.078+0000] {processor.py:842} WARNING - No viable dags retrieved from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:32:49.096+0000] {processor.py:183} INFO - Processing /root/airflow/dags/financial_trading_dag.py took 0.029 seconds
[2025-04-16T20:33:19.418+0000] {processor.py:161} INFO - Started process (PID=39) to work on /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:33:19.419+0000] {processor.py:830} INFO - Processing file /root/airflow/dags/financial_trading_dag.py for tasks to queue
[2025-04-16T20:33:19.421+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:33:19.420+0000] {dagbag.py:545} INFO - Filling up the DagBag from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:33:19.423+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:33:19.422+0000] {dagbag.py:355} ERROR - Failed to import: /root/airflow/dags/financial_trading_dag.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/airflow/dags/financial_trading_dag.py", line 6, in <module>
    from src.data.get_stock import download_stock_data
ModuleNotFoundError: No module named 'src'
[2025-04-16T20:33:19.424+0000] {processor.py:842} WARNING - No viable dags retrieved from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:33:19.438+0000] {processor.py:183} INFO - Processing /root/airflow/dags/financial_trading_dag.py took 0.025 seconds
[2025-04-16T20:33:49.786+0000] {processor.py:161} INFO - Started process (PID=40) to work on /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:33:49.797+0000] {processor.py:830} INFO - Processing file /root/airflow/dags/financial_trading_dag.py for tasks to queue
[2025-04-16T20:33:49.798+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:33:49.798+0000] {dagbag.py:545} INFO - Filling up the DagBag from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:33:49.800+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:33:49.800+0000] {dagbag.py:355} ERROR - Failed to import: /root/airflow/dags/financial_trading_dag.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/airflow/dags/financial_trading_dag.py", line 6, in <module>
    from src.data.get_stock import download_stock_data
ModuleNotFoundError: No module named 'src'
[2025-04-16T20:33:49.801+0000] {processor.py:842} WARNING - No viable dags retrieved from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:33:49.816+0000] {processor.py:183} INFO - Processing /root/airflow/dags/financial_trading_dag.py took 0.035 seconds
[2025-04-16T20:34:20.906+0000] {processor.py:161} INFO - Started process (PID=41) to work on /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:34:20.907+0000] {processor.py:830} INFO - Processing file /root/airflow/dags/financial_trading_dag.py for tasks to queue
[2025-04-16T20:34:20.908+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:34:20.908+0000] {dagbag.py:545} INFO - Filling up the DagBag from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:34:20.910+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:34:20.910+0000] {dagbag.py:355} ERROR - Failed to import: /root/airflow/dags/financial_trading_dag.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/airflow/dags/financial_trading_dag.py", line 6, in <module>
    from src.data.get_stock import download_stock_data
ModuleNotFoundError: No module named 'src'
[2025-04-16T20:34:20.911+0000] {processor.py:842} WARNING - No viable dags retrieved from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:34:20.926+0000] {processor.py:183} INFO - Processing /root/airflow/dags/financial_trading_dag.py took 0.024 seconds
[2025-04-16T20:35:12.058+0000] {processor.py:161} INFO - Started process (PID=27) to work on /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:35:12.060+0000] {processor.py:830} INFO - Processing file /root/airflow/dags/financial_trading_dag.py for tasks to queue
[2025-04-16T20:35:12.061+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:35:12.061+0000] {dagbag.py:545} INFO - Filling up the DagBag from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:35:12.067+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:35:12.066+0000] {dagbag.py:355} ERROR - Failed to import: /root/airflow/dags/financial_trading_dag.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/airflow/dags/financial_trading_dag.py", line 6, in <module>
    from src.data.get_stock import download_stock_data
ModuleNotFoundError: No module named 'src.data.get_stock'
[2025-04-16T20:35:12.067+0000] {processor.py:842} WARNING - No viable dags retrieved from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:35:12.084+0000] {processor.py:183} INFO - Processing /root/airflow/dags/financial_trading_dag.py took 0.030 seconds
[2025-04-16T20:35:42.177+0000] {processor.py:161} INFO - Started process (PID=28) to work on /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:35:42.178+0000] {processor.py:830} INFO - Processing file /root/airflow/dags/financial_trading_dag.py for tasks to queue
[2025-04-16T20:35:42.179+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:35:42.179+0000] {dagbag.py:545} INFO - Filling up the DagBag from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:35:42.181+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:35:42.181+0000] {dagbag.py:355} ERROR - Failed to import: /root/airflow/dags/financial_trading_dag.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/airflow/dags/financial_trading_dag.py", line 6, in <module>
    from src.data.get_stock import download_stock_data
ModuleNotFoundError: No module named 'src.data.get_stock'
[2025-04-16T20:35:42.182+0000] {processor.py:842} WARNING - No viable dags retrieved from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:35:42.197+0000] {processor.py:183} INFO - Processing /root/airflow/dags/financial_trading_dag.py took 0.024 seconds
[2025-04-16T20:36:12.351+0000] {processor.py:161} INFO - Started process (PID=29) to work on /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:36:12.352+0000] {processor.py:830} INFO - Processing file /root/airflow/dags/financial_trading_dag.py for tasks to queue
[2025-04-16T20:36:12.353+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:36:12.353+0000] {dagbag.py:545} INFO - Filling up the DagBag from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:36:12.357+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:36:12.356+0000] {dagbag.py:355} ERROR - Failed to import: /root/airflow/dags/financial_trading_dag.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/airflow/dags/financial_trading_dag.py", line 6, in <module>
    from src.data.get_stock import download_stock_data
ModuleNotFoundError: No module named 'src.data.get_stock'
[2025-04-16T20:36:12.357+0000] {processor.py:842} WARNING - No viable dags retrieved from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:36:12.373+0000] {processor.py:183} INFO - Processing /root/airflow/dags/financial_trading_dag.py took 0.027 seconds
[2025-04-16T20:36:43.401+0000] {processor.py:161} INFO - Started process (PID=30) to work on /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:36:43.411+0000] {processor.py:830} INFO - Processing file /root/airflow/dags/financial_trading_dag.py for tasks to queue
[2025-04-16T20:36:43.413+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:36:43.413+0000] {dagbag.py:545} INFO - Filling up the DagBag from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:36:43.417+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:36:43.416+0000] {dagbag.py:355} ERROR - Failed to import: /root/airflow/dags/financial_trading_dag.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/airflow/dags/financial_trading_dag.py", line 6, in <module>
    from src.data.get_stock import download_stock_data
ModuleNotFoundError: No module named 'src.data.get_stock'
[2025-04-16T20:36:43.417+0000] {processor.py:842} WARNING - No viable dags retrieved from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:36:43.433+0000] {processor.py:183} INFO - Processing /root/airflow/dags/financial_trading_dag.py took 0.036 seconds
[2025-04-16T20:37:16.519+0000] {processor.py:161} INFO - Started process (PID=31) to work on /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:37:16.529+0000] {processor.py:830} INFO - Processing file /root/airflow/dags/financial_trading_dag.py for tasks to queue
[2025-04-16T20:37:16.530+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:37:16.530+0000] {dagbag.py:545} INFO - Filling up the DagBag from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:37:16.533+0000] {logging_mixin.py:188} INFO - [2025-04-16T20:37:16.533+0000] {dagbag.py:355} ERROR - Failed to import: /root/airflow/dags/financial_trading_dag.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/airflow/dags/financial_trading_dag.py", line 6, in <module>
    from src.data.get_stock import download_stock_data
ModuleNotFoundError: No module named 'src.data.get_stock'
[2025-04-16T20:37:16.533+0000] {processor.py:842} WARNING - No viable dags retrieved from /root/airflow/dags/financial_trading_dag.py
[2025-04-16T20:37:16.549+0000] {processor.py:183} INFO - Processing /root/airflow/dags/financial_trading_dag.py took 0.035 seconds
